assets/javascripts/discourse/initializers/initialize-group-tracker.js

```diff
@@ -33,20 +35,22 @@ function addTrackedGroupToTopicList(api) {
   );
 
   withSilencedDeprecations("discourse.hbr-topic-list-overrides", () => {
-    api.modifyClass("component:topic-list-item", {
-      pluginId: PLUGIN_ID,
-
-      @computed("topic.first_tracked_post")
-      unboundClassNames(firstTrackedPost) {
-        let classNames = this._super();
-
-        if (firstTrackedPost) {
-          classNames += ` group-${firstTrackedPost.group}`;
+    api.modifyClass(
+      "component:topic-list-item",
+      (Superclass) =>
+        class extends Superclass {
+          @computed("topic.first_tracked_post")
+          unboundClassNames(firstTrackedPost) {
+            let classNames = super.unboundClassNames();
```

Comment:
shouldn't this be:

```suggestion
            let classNames = super.unboundClassNames;
```

?